core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -243,6 +250,34 @@ object BootMonitors extends LazyLogging {
       ))
   }
 
+  private def startWorkspaceMigrationActor(system: ActorSystem,
+                                           config: Config,
+                                           credential: Credential,
+                                           dataSource: SlickDataSource,
+                                           workspaceService: UserInfo => WorkspaceService,
+                                           storageService: GoogleStorageService[IO],
+                                           storageTransferService: GoogleStorageTransferService[IO],
+                                           sam: SamDAO) = {
+    val serviceProject = GoogleProject(config.getConfig("gcs").getString("serviceProject"))
+    val rawlsUserInfo = UserInfo.buildFromTokens(credential)
+
+    if (Try(config.getBoolean("enableWorkspaceMigrationActor")) == Success(true)) {
+      system.spawn(
+        WorkspaceMigrationActor(
+          pollingInterval = 10.seconds,
```

Comment:
In the past I think we have put these values in config, is there a reason not to do that here?

Reply:
We might, I just didn't do it in this changelist and I don't know how to do config stuff. I'll create a ticket.

Reply:
https://broadworkbench.atlassian.net/browse/CA-1807